### PR TITLE
upgrade dependencies, removes ios/android builds, bug fixes

### PR DESCRIPTION
**Pull request by vickiwong85**
# Problem
ios and android build files were interrupting eas build
building locally also fails due to old dependencies
other bugs:
- android settings button not working after initial swipe (upgrading to new dependencies gave me new warning to fix the issue)
- fixing font scaling with new degree text styles

# Solution:
- remove android/ios folders and added to .gitignore
- update dependencies and fix any resulting errors
- update README with build instructions
- fix font scaling for ThermostatGauge degree and text

Android showing largest font and iOS showing regular font:
Screenshot 2024-02-19 at 4 11 04 PM

### CODE REVIEW

**Comment by mia-lorraine** [comment]
i suggest squashing all commits before merge c:

File: yarn.lock [hunk 1]
```diff
@@ -4388,231 +5682,193 @@ execa@^1.0.0:
...
+  version "3.1.2"
+  resolved "https://registry.yarnpkg.com/expo-structured-headers/-/expo-structured-headers-3.1.2.tgz#ac2ec100349a398f5d4fcc450d9b963b9a6efcff"
+  integrity sha512-UjVKwKeq6Ya4FWweDyfeaB2a2FY6BzrgF/c2uRMUDRVbUZIkhmc6BNYHKmfS2bnYWOc3bqp/REX5YFLSQfqvOg==
+
+expo-updates-interface@~0.9.0:
+  version "0.9.3"
+  resolved "https://registry.yarnpkg.com/expo-updates-interface/-/expo-updates-interface-0.9.3.tgz#32caf6fe48b517c904be901dac46d2c964ca8407"
+  integrity sha512-zBkJmLP5Bmcz2s14PZDcHFzRjnyiG/TwlQfIvpnV4NS+EYyR4vgbC0J3zUEi7Vvhm5m4gjsTpU6X1oKn6br+HQ==
+
+expo-updates@~0.16.4:
```
Comment (mia-lorraine) [inline yarn.lock:5811]: [expo-updates](https://github.com/ACE-IoT-Solutions/connecting-mha-app/pull/5) addressed here

File: yarn.lock [hunk 2]
```diff
@@ -2331,23 +3196,28 @@
     hoist-non-react-statics "^3.3.2"
     react-native-safe-area-view "^0.14.9"
 
-"@react-navigation/native@^6.0.6":
-  version "6.0.10"
-  resolved "https://registry.npmjs.org/@react-navigation/native/-/native-6.0.10.tgz"
-  integrity sha512-H6QhLeiieGxNcAJismIDXIPZgf1myr7Og8v116tezIGmincJTOcWavTd7lPHGnMMXaZg94LlVtbaBRIx9cexqw==
+"@react-navigation/native@^6.1.7":
```
Comment (mia-lorraine) [inline yarn.lock:3199]: [react-navigation/native](https://github.com/ACE-IoT-Solutions/connecting-mha-app/pull/4) addressed

File: yarn.lock [hunk 3]
```diff
@@ -7348,21 +8806,21 @@ rc@~1.2.7:
...
+  integrity sha512-cq/o30z9W2Wb4rzBefjv5fBalHU0rJGZCHAkf/RHSBWSSYwh8PlQTqqOJmgIIbBtpj27T6FIPXeomIjZtCNVqA==
   dependencies:
     shell-quote "^1.6.1"
     ws "^7"
 
-react-dom@18.0.0:
-  version "18.0.0"
-  resolved "https://registry.yarnpkg.com/react-dom/-/react-dom-18.0.0.tgz#26b88534f8f1dbb80853e1eabe752f24100d8023"
-  integrity sha512-XqX7uzmFo0pUceWFCt7Gff6IyIMzFUn7QMZrbrQfGxtaxXZIcGQzoNpRLE3fQLnS4XzLLPMZX2T9TRcSrasicw==
+react-dom@18.2.0:
```
Comment (mia-lorraine) [inline yarn.lock:8817]: [PR addressed](https://github.com/ACE-IoT-Solutions/connecting-mha-app/pull/1)

File: yarn.lock [hunk 4]
```diff
@@ -1523,23 +2019,23 @@
...
-  version "0.3.3"
-  resolved "https://registry.yarnpkg.com/@expo/cli/-/cli-0.3.3.tgz#797ae9605c0556ba5159f5fbe9844a4e71380fcc"
-  integrity sha512-5uDbjLntXkJOPx2SOZ6Pj9MC448YWtGenBJE+jNBfAbUPaf62Fy9uIyWWeKeRWXNfSs9jmOVpIgyYLTklV1vpg==
-  dependencies:
-    "@babel/runtime" "^7.14.0"
-    "@expo/code-signing-certificates" "^0.0.2"
-    "@expo/config" "~7.0.1"
-    "@expo/config-plugins" "~5.0.1"
-    "@expo/dev-server" "0.1.120"
+"@expo/cli@0.7.3":
```
Comment (mia-lorraine) [inline yarn.lock:2022]: [expo cli update](https://github.com/ACE-IoT-Solutions/connecting-mha-app/pull/6) to 0.7.3

File: package.json [hunk 5]
```diff
@@ -22,32 +22,33 @@
   },
   "dependencies": {
     "@expo-google-fonts/rubik": "^0.2.0",
-    "@react-native-async-storage/async-storage": "1.17.11",
+    "@react-native-async-storage/async-storage": "1.21.0",
     "@react-native-community/slider": "4.4.2",
-    "@react-native-picker/picker": "2.4.8",
-    "@react-navigation/bottom-tabs": "^6.0.9",
+    "@react-native-picker/picker": "2.6.1",
+    "@react-navigation/bottom-tabs": "^6.5.7",
```
Comment (vickiwong85) [inline package.json:28]: [bottom-tabs update](#3) addressed here

File: package.json [hunk 6]
```diff
@@ -22,32 +22,33 @@
...
     "@react-navigation/native": "^6.1.7",
-    "@react-navigation/native-stack": "^6.2.5",
+    "@react-navigation/native-stack": "^6.9.12",
```
Comment (vickiwong85) [inline package.json:30]: [native stack upgrade](#2 ) addressed here

**Review by mia-lorraine — CHANGES_REQUESTED** [review]
imo, would make a new PR to show files changed and squash commits for a cleaner merge c:

**Pull request closed without merging**